schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -84,6 +87,52 @@
       ],
       "description": "Microsoft.Sql/servers"
     },
+    "servers_administrators": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
```

Comment:
If it's just string, you don't need the option for expression. You can remove the "oneOf".